dvc/remote/local.py

```diff
@@ -626,8 +626,12 @@ def _process(
             func = pbar.wrap_fn(func)
             with ThreadPoolExecutor(max_workers=jobs) as executor:
                 if download:
-                    fails = sum(executor.map(func, *dir_plans))
-                    fails += sum(executor.map(func, *file_plans))
+                    from_infos, to_infos, names, _ = (
+                        d + f for d, f in zip(dir_plans, file_plans)
+                    )
+                    fails = sum(
+                        executor.map(func, from_infos, to_infos, names)
+                    )
                 else:
```

Comment:
`_get_plans` returns four values (previously 3, with the new one being checksum).
https://github.com/iterative/dvc/blob/4d140a99445cbdcdf020b43551dc2aee9f970686/dvc/remote/local.py#L577

This checksum was received by `download` as `no_progress_bar` args.
https://github.com/iterative/dvc/blob/4d140a99445cbdcdf020b43551dc2aee9f970686/dvc/remote/base.py#L330-L338